test01/test01/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace test01
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Солнышко");
+
+            Console.WriteLine();
+
+            Console.ForegroundColor= ConsoleColor.Yellow;
+            Console.WriteLine("Вышло солнце в небеса\r\nИ глядит во все глаза.\r\nЗдравствуй, солнце золотое!\r\nХорошо гулять с тобою -\r\nСловно добрыми руками,\r\nОбнимаешь нас лучами.");
```

Comment:
Если весь текст стихотворения помещать в один оператор (выводить одной строковой константой), символ `\r` возврата каретки лишний. Достаточно использовать перевод строки `\n`.
```suggestion
            Console.WriteLine("Вышло солнце в небеса\nИ глядит во все глаза.\nЗдравствуй, солнце золотое!\nХорошо гулять с тобою -\nСловно добрыми руками,\nОбнимаешь нас лучами.");
```